libs/sdk-backend-tiger/tests/integrated/backend.ts

```diff
@@ -66,5 +72,115 @@ export function testBackend(): IAnalyticalBackend {
 
 export function testWorkspace(): string {
     // UI SDK Reference workspace ID
-    return "4dc4e033e611421791adea58d34d958c";
+    const workspace: string = process.env.WORKSPACE_ID ?? "4aefdef78922461faeb08c12de183f81";
+    return workspace;
+}
+
+export function compareJsonDisregardTheOrder(jsonActual: any, jsonExpected: any) {
+    if (Object.prototype.toString.call(jsonActual) !== Object.prototype.toString.call(jsonExpected)) {
+        return false;
+    }
+
+    if (Array.isArray(jsonActual)) {
+        if (jsonActual.length !== jsonExpected.length) {
+            return false;
+        }
+
+        let i = jsonActual.length;
+        while (i--) {
+            let j = jsonExpected.length;
+            let found = false;
+
+            while (!found && j--) {
+                if (compareJsonDisregardTheOrder(jsonActual[i], jsonExpected[j])) found = true;
+            }
+
+            if (!found) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    if (Object.prototype.toString.call(jsonActual) === "[object Object]") {
+        for (const key in jsonActual) {
+            if (Object.prototype.hasOwnProperty.call(jsonActual, key)) {
+                if (!Object.prototype.hasOwnProperty.call(jsonExpected, key)) {
+                    return false;
+                }
+
+                if (!compareJsonDisregardTheOrder(jsonActual[key], jsonExpected[key])) {
+                    return false;
+                }
+            }
+        }
+
+        for (const key in jsonExpected) {
+            if (
+                Object.prototype.hasOwnProperty.call(jsonExpected, key) &&
+                !Object.prototype.hasOwnProperty.call(jsonActual, key)
+            ) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    return jsonActual === jsonExpected;
+}
+
+export async function sanitizeKeyWithNewValue(result: any, key: string, newValue: string) {
+    const fixtureContent = JSON.stringify(result);
```

Comment:
What about if we just use some regex, to replace workspace everywhere in the result with something else (e.g. test-workspace string)? Which should be easier to forward? In a similar way as we have it in `mock-handling` https://github.com/gooddata/gooddata-ui-sdk/commit/334ce71cbc9adfd3ab88a6b37591d968f3af2cf6#diff-e02ed405440d5de12fa90da0717a09474f95a0fc566e2652266231c4724b0337R79

Then you also don't need to delete the workspace property in sanitizeWorkspace, every workspace id will be replaced with the sanitized value.

Reply:
use regex to replace workspaceId, instead of delete it